dhall-lsp-server/src/LSP/Handlers/DocumentFormatting.hs

```diff
@@ -6,23 +6,19 @@ import qualified Backend.Dhall.Formatting as Formatting
 
 import qualified Language.Haskell.LSP.Core as LSP.Core
 
-import qualified Data.Aeson                            as J
 import qualified Language.Haskell.LSP.Types            as J
-import qualified Language.Haskell.LSP.Types.Lens       as J
 import qualified Language.Haskell.LSP.Utility  as LSP.Utility
-import           Language.Haskell.LSP.Messages
 
 import qualified Data.Text
 import qualified Data.Text.IO
-import qualified Data.SortedList
 import Control.Monad.Trans (lift)
 import Control.Monad.Reader (ReaderT)
 
 -- TODO: implement tabSize and spaces/tabs options
 -- * Note: any formatting errors would be swallowed. I think this is fine in this case, but generally we'd like to send user a notification
 -- (e.g. the error occurred in the formatter itself, and user requests format constantly and nothing happens)
 formatDocument :: J.Uri -> Int -> Bool -> ReaderT (LSP.Core.LspFuncs ()) IO (J.List J.TextEdit)
-formatDocument fileUri tabSize insertSpaces = do
+formatDocument fileUri _ {-tabSize-} _ {-insertSpaces-} = do
```

Comment:
If you commented this out to avoid warnings about unused bindings, then you can just prefix names with underscores: 
```suggestion
formatDocument fileUri _tabSize _insertSpaces = do
```

Reply:
Didn't know that! Thanks!